crates/lightbeam/src/module.rs

```diff
@@ -642,7 +644,9 @@ pub fn translate_only(data: &[u8]) -> Result<TranslatedModule, Error> {
         translate_sections::data(data)?;
     }
 
-    assert!(reader.eof());
+    if !reader.eof() {
+        return Err(Error::Input("Module not transate completely".to_string()));
```

Comment:
This wording is a little unclear; perhaps this could say "Unexpected data found after the end of the module" or so.

Reply:
you right it's more clear